src/layers.json

```diff
@@ -7,7 +7,8 @@
       "python2.7": "arn:aws:lambda:af-south-1:464622532012:layer:Datadog-Python27:29",
       "python3.6": "arn:aws:lambda:af-south-1:464622532012:layer:Datadog-Python36:29",
       "python3.7": "arn:aws:lambda:af-south-1:464622532012:layer:Datadog-Python37:29",
-      "python3.8": "arn:aws:lambda:af-south-1:464622532012:layer:Datadog-Python38:29"
+      "python3.8": "arn:aws:lambda:af-south-1:464622532012:layer:Datadog-Python38:29",
+      "extension": "arn:aws:lambda:af-south-1:464622532012:layer:Datadog-Extension:5"
```

Comment:
The modification to the generate_layers.sh script generated the addition of the extension arns in the layers.json file however the script was not run to generate the extension arns for the layer-gov.json file (for gov cloud).

Reply:
I was unable to generate those arns via the script because I do not have gov cloud access.

Reply:
I think that should be fine as long as the next person who releases the plugin has GovCloud access. Are you confident that the script will work for GovCloud for someone who does have access?

Reply:
The publish_prod.sh script checks if the user wants to update the layers while publishing a new version to production [here](https://github.com/DataDog/serverless-plugin-datadog/blob/41a10c6a67039218969c68bd56ebadae167825a8/scripts/publish_prod.sh#L50). Inside there the user needs to have gov cloud and prod-engineering access to run the generate_layers_json script. I am confident that if someone who has gov cloud access ran the publish script while also wishing to update the layers then both regular and gov cloud arns should be generated.

Reply:
There may be a limbo period where gov cloud customers wishing to use the extension may not be able to until someone updates those layers.

Reply:
I don't think there would be a gap because the changes in this PR won't be available until someone releases a new version of this plugin, which would entail updating the layers, right?